admin_guide/high_availability.adoc

```diff
@@ -580,6 +588,51 @@ the list of VIPs.  The vrrp-ids may have values in the range 1..255.
 
 When there are multiple ipfailover deployment configuration care must be taken to specify `--vrrp-id-offset` so that there is room to increase the number of VIPS in the deployment configuration and none of the vrrp-id ranges overlap.
 
+[[admin-guide-high-availability-configuring-more-than-254]]
+=== Configuring IP failover for more than 254 addresses
+
+By default, IP failover has a limitation of managing only 254 VIP addresses. You
+can group sets of VIPs for VRRP instances using the `virtual-ip-groups` option
+to define the number of VIP groups available for each VRRP instance when
+cxref:../admin_guide/high_availability.adoc#configuring-ip-failover[configuring
+IP failover]. 
+
+This creates a wider range of allocation of VIPs per VRRP in the case of VRRP
+failover events, and is useful when all hosts in the cluster have access to a
+service locally. For example, when a service is being exposed with an external
+IP. 
+
+[NOTE]
+====
+As a rule for failover, services, such as the router, should not be limited to
+one specific host. Instead, services should be replicated to each host so that
+in the case of IP failover, the services do not have to be recreated on the new
+host.
+
+If you are using {product-title} health checks, the nature of IP failover and
```

Comment:
Is this really the same note? Should this be a separate limitation?
"If you use {product-title} health checks, do not configure IP failover for groups. Not all instances in the group can be checked for <more detail about what's checked or failing> during failover. You can link:<link to configuring K8 health checks>[configure Kubernetes health checks] for these groups instead."